docs/home/howto/PATCHING.md

```diff
@@ -0,0 +1,79 @@
+# Patching OS with running Epiphany components
+This guide describes steps you have to perform to patch RedHat systems in a way to don't interrupt working Epiphany components.
+
+### Disclamer
+We providing a recommended way to patch your RedHat systems. Before proceeding with patching the production environment we highly recommending performing an update of your test cluster first.  
+This document will help you decide how you should patch your OS. This is not a step-by-step guide.
+
+### Requirements:
+- The fresh, actual backup contains your all sensitive data
+
+## AWS
+### Suggested OS images
+For current Epiphany version we recommend the following image (AMI):  
+`RHEL-7.8_HVM_GA-20200225-x86_64-1-Hourly2-GP2` which contain `3.10.0-1127.el7.x86_64` kernel version.
+
+> Note: For different supported OS versions this guide may be useful as well.
+### Patching methods
+AWS provides `Patch Manager` for patching OSes. This option is available via:  
+[Systems Manager](https://eu-central-1.console.aws.amazon.com/systems-manager/) > Instances & Nodes > [Patch Manager](https://eu-central-1.console.aws.amazon.com/systems-manager/patch-manager) and it has equivalent in [aws cli](https://docs.aws.amazon.com/systems-manager/latest/userguide/patch-manager-cli-commands.html).  
+To use AWS Patch Manager please use the following instruction: [link](https://docs.aws.amazon.com/systems-manager/latest/userguide/patch-manager-how-it-works.html)  
+This feature will let you automate patching your instances.  
+Benefits:
+```shell
+- Automate Patching
+- Define approval rules
+- Create patch baselines
+- Monitor compliance
+```
+We won't detail these aspects in this doc because they were already well covered by AWS in the above link.
+
+AWS supports a bunch of paid solutions to perform kernel patching without system reboot.  
+Examples: RedHat [kpatch](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/kernel_administration_guide/applying_patches_with_kernel_live_patching) or external tool from its partner [KernelCare](https://www.kernelcare.com/) which is available in [AWS Marketplace](https://aws.amazon.com/marketplace/pp/B085ZLFK7B) in SaaS model.  
+If you have a valid subscription for any of the above tools, we highly recommend using it to patch your system.
+
+## Azure
+### Suggested OS images
+For current Epiphany version we recommend the following image (urn):  
```

Comment:
Make it explicit which you mean, in case we missed updating this document during releases.